autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillManagementRecyclerAdapter.kt

```diff
@@ -25,41 +25,66 @@ import androidx.recyclerview.widget.RecyclerView.Adapter
 import com.duckduckgo.app.browser.favicon.FaviconManager
 import com.duckduckgo.autofill.domain.app.LoginCredentials
 import com.duckduckgo.autofill.impl.databinding.ItemRowAutofillCredentialsManagementScreenBinding
+import com.duckduckgo.autofill.impl.databinding.ItemRowAutofillCredentialsManagementScreenHeaderBinding
 import kotlinx.coroutines.launch
 
 class AutofillManagementRecyclerAdapter(
     val lifecycleOwner: LifecycleOwner,
     val faviconManager: FaviconManager,
+    val grouper: CredentialGrouper,
     val onCredentialSelected: (credentials: LoginCredentials) -> Unit,
     val onCopyUsername: (credentials: LoginCredentials) -> Unit,
     val onCopyPassword: (credentials: LoginCredentials) -> Unit
-) : Adapter<AutofillManagementRecyclerAdapter.CredentialsViewHolder>() {
+) : Adapter<RecyclerView.ViewHolder>() {
```

Comment:
Later when we move away from notifydatasetchange we can use ListAdapter instead.